Starting job at Fri Feb 14 10:05:58 AM CET 2025
Running Filter_by_part.py for subdirectory 22010, part 12
2025-02-14 10:06:02 - INFO - Event filtering process starts...
2025-02-14 10:06:02 - INFO - Initialized PureNeutrinoEventFilter
2025-02-14 10:06:02 - INFO - Starting filtering process for 22010/12
2025-02-14 10:06:04 - INFO - Loaded 10 event specifier files from /lustre/hpc/project/icecube/HE_Nu_Aske_Oct2024/clean_events_dict/22010/22010/reduced
2025-02-14 10:06:04 - ERROR - An error occurred: 'Field "RunID" does not exist in schema'
Traceback (most recent call last):
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 85, in <module>
    main()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 81, in main
    run()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/Filter_by_part.py", line 72, in run
    event_filter()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/PureNeutrinoEventFilter.py", line 23, in __call__
    self._filter_truth()
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/EventFilter.py", line 129, in wrapper
    return method(*args, **kwargs)
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/PureNeutrinoEventFilter.py", line 33, in _filter_truth
    relevant_csvs = self._get_relevant_csv_files(truth_table)
  File "/lustre/hpc/icecube/cyan/factory/DOMification/Selection/PureNeutrinoEventFilter.py", line 67, in _get_relevant_csv_files
    run_ids = truth_table.column("RunID").to_pylist()
  File "pyarrow/table.pxi", line 1779, in pyarrow.lib._Tabular.column
  File "pyarrow/table.pxi", line 1715, in pyarrow.lib._Tabular._ensure_integer_index
KeyError: 'Field "RunID" does not exist in schema'
Job completed at Fri Feb 14 10:06:04 AM CET 2025
